File: bus/cus/cus-entity/src/main/java/com/drzyl/cus/entity/CusAgent.java
package com.drzyl.cus.entity;

import java.io.Serializable;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;

import lombok.Data;

/**
 * 客户委托信息 BUS_CUS_AGENT 实体类
 */
@Data
@TableName(value = "BUS_CUS_AGENT")
public class CusAgent implements Serializable {

  private static final long serialVersionUID = 1L;

  @TableId
  protected String cusAgentId; // ID

  protected String cusBaseId; // 客户ID

  protected String aePer; // 委托代理人

  protected String aePhone; // 委托代理人电话

  protected String aeFax; // 委托代理人传真

  protected String aeMatter; // 委托事项

  protected String aeDate; // 委托时间

  protected String orgId; // 机构ID

  protected String aeEdate; // 委托结束时间

  protected String uploadIds; // 附件IDs

  protected String remark; // 备注

  protected String actProcessInstanceId; // 流程实例ID

  protected String actProcessStatus; // 流程状态

  protected Integer versions; // 版本号

  protected String createId; // 创建人账号

  protected String createName; // 创建人名称

  protected String createOrg; // 创建机构

  protected Long createTimes; // 创建时间

  protected String modifyId; // 修改人账号

  protected String modifyName; // 修改人名称

  protected String modifyOrg; // 修改机构

  protected Long modifyTimes; // 修改时间

  protected String delFlag; // 逻辑删除0=删除,1=显示

}

File: vue-coalsale/src/view/com/drzyl/cus/cusAgent/CusAgent.js
export default class CusAgent {
  constructor () {
    this.cusAgentId = null // ID
    this.cusBaseId = null // 客户ID
    this.aePer = null // 委托代理人
    this.aePhone = null // 委托代理人电话
    this.aeFax = null // 委托代理人传真
    this.aeMatter = null // 委托事项
    this.aeDate = null // 委托时间
    this.orgId = null // 机构ID
    this.aeEdate = null // 委托结束时间
    this.uploadIds = null // 附件IDs
    this.remark = null // 备注
    this.actProcessInstanceId = null // 流程实例ID
    this.actProcessStatus = null // 流程状态
    this.versions = null // 版本号
    this.createId = null // 创建人账号
    this.createName = null // 创建人名称
    this.createOrg = null // 创建机构
    this.createTimes = null // 创建时间
    this.modifyId = null // 修改人账号
    this.modifyName = null // 修改人名称
    this.modifyOrg = null // 修改机构
    this.modifyTimes = null // 修改时间
    this.delFlag = null // 逻辑删除0=删除,1=显示
  }
}
